bigbluebutton-html5/imports/ui/components/shortcut-help/service.jsx

```diff
@@ -10,15 +10,15 @@ const withShortcutHelper = (WrappedComponent, param) => (props) => {
   if (ENABLED_SHORTCUTS) {
     shortcuts = Object.values(BASE_SHORTCUTS).map((el) => {
       const obj = { ...el };
-      obj.descIdLowerCase = obj.descId.toLowerCase();
+      obj.descId = obj.descId.toLowerCase();
       return obj;
     }).filter(el => ENABLED_SHORTCUTS.includes(el.descIdLowerCase));
   }
 
   if (param !== undefined) {
     if (!Array.isArray(param)) {
       shortcuts = shortcuts
-        .filter(el => el.descIdLowerCase === param.toLowerCase())
+        .filter(el => el.descId.toLowerCase() === param.toLowerCase())
```

Comment:
Didn't you set `descId` to already be lower case on line 13?

Reply:
only if `ENABLED_SHORTCUTS`

Reply:
was originally added here https://github.com/bigbluebutton/bigbluebutton/commit/c4462c4cb0c3cfc17d96413afddc2c74d8251dd5#diff-be541319d8f54280f6cbfe3b54edcaabR21